buildscripts/validate-vendor.sh

```diff
@@ -23,6 +23,8 @@ if [ -d vendor ]; then
   go mod vendor
 fi
 
+chmod 644 "go.sum"
```

Comment:
This must be a bug. Even if `go mod ...` updates the contents, it should not change file permissions

Reply:
Yea copied the issue here: https://github.com/golang/go/issues/34634 I think it's resolved in newer versions of go but this is on 1.12

Reply:
With go 1.12 having reached EOL, should we update to go 1.13 or go 1.14 @justincormack ? I see there's some existing PRs but they appear to be outdated; https://github.com/theupdateframework/notary/pull/1515, https://github.com/theupdateframework/notary/pull/1522, https://github.com/theupdateframework/notary/pull/1536

Reply:
Could this potentially be merged in to get get updated dependencies working for 1.12 until we can migrate to a newer version of go?

Reply:
I would rather update Go first.